darts/tests/models/components/test_layer_norm_variants.py

```diff
@@ -25,3 +28,37 @@ def test_lnv(self):
                 ln = layer_norm(4)
                 inputs = torch.zeros(1, 4, 4)
                 ln(inputs)
+
+        def test_rin(self):
+
+            x = torch.randn(3, 4, 7)
```

Comment:
this test is subjected to randomness and can eventually fail. Can we address this?

Reply:
I added RNG seeding for torch and numpy.